tests/models/test_representations.py

```diff
@@ -41,8 +42,10 @@ def setUp(self):
         self.test_model = product_root.add_model(name=self.test_model_name, multiplicity=Multiplicity.ONE)
         self.prop_model = self.test_model.add_property(
             name=self.test_prop_name,
-            type=self.property_type,
+            property_type=self.property_type,
         )  # type: AnyProperty
+        if self.options:
+            self.prop_model.options = self.options
```

Comment:
I would modify the sub-class of test specifically for the single-select property if the options have to be set. Now you're adding a work-around to this superclass instead. Simplify create another setUp() method that calls super().setUp(), but with the options attached.

Reply:
You're right, I fixed it now